node-sdk-sample/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sdk-sample",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "commonjs",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint src",
+    "build": "tsc",
+    "watch": "tsc -w"
+  },
+  "author": "Zowe",
+  "license": "EPL-2.0",
+  "description": "Zowe Client Node SDK Sample",
+  "devDependencies": {
+    "@types/jest": "^29.5.14",
+    "@typescript-eslint/eslint-plugin": "^8.19.1",
+    "@typescript-eslint/parser": "^8.19.1",
+    "@zowe/imperative": "^8.10.4",
+    "eslint": "^8.57.1",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.5",
+    "tsx": "^4.19.2",
```

Comment:
if we replace the `npx tsx src/script.ts`, we should be able to remove this dependency.